src/utrition/src/pages/upload_page.js

```diff
@@ -14,7 +14,7 @@ const Upload = () => {
         height: "90vh",
       }}
     >
-      <h1 style={{ clear: "right" }}>Upload </h1>
+      <h1 style={{borderBottom: "2mm ridge rgba(60, 179, 113, .7)", position: "absolute", top:"30%", clear: "right" }}>Upload an Image</h1>
```

Comment:
Remove border bottom so line of code looks like this instead
`<h1 style={{position: "absolute", top:"30%", clear: "right" }}>Upload an Image</h1>`